sdk/src/test/java/io/radar/sdk/RadarTrackingOptionsTest.kt

```diff
@@ -0,0 +1,17 @@
+package io.radar.sdk
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+class RadarTrackingOptionsTest {
+
+    @Test
+    fun testEquals() {
+        assertEquals(RadarTrackingOptions.EFFICIENT, RadarTrackingOptions.fromJson(RadarTrackingOptions.EFFICIENT.toJson()))
```

Comment:
this is a nice test, but not really related to the PR

Reply:
anyways, minor nit

Reply:
Yeah this was added to test out (and find) a bug with differences in the RadarTrackingOptions between client and server (@jsani-radar did a followup PR in schemes after this stuff)